db/migrate/20180105205907_allow_partial_signup_state.rb

```diff
@@ -0,0 +1,8 @@
+class AllowPartialSignupState < ActiveRecord::Migration
+  def change
+    change_column_null :signup_states, :contact_info_value, true
+    change_column_null :signup_states, :contact_info_kind, true
+
+    add_column :signup_states, :is_partial_info_allowed, :boolean, default: false, null: false
+  end
```

Comment:
I'm not sure if we need to store `is_partial_info_allowed`.  It could just be an `attr_accessor` for so the validation could read it while saving.

However it's just a bool column and doesn't hurt anything really